lib/pbench/agent/tool_data_sink.py

```diff
@@ -980,19 +1136,25 @@ def main(argv):
         logger.error("External 'tar' executable not found")
         return 2
 
+    if not find_executable("podman"):
+        logger.error(
+            "Podman is not installed on this system (required by some tools, aborting launch)"
+        )
+        return 3
```

Comment:
Here we have exit status chaos again; changing these means we'll never know what they mean without tracking down the exact source code running on some system. (Tracking all the agent tool meister versions helps, some, but it's still a tedious debugging mess.) Some discussion of this in #2005 but no grand unified exist status theory yet. Still, we should keep in mind that every time we change a status code, we're potentially complicating future triage.

Reply:
Agreed.  I'd rather not address this problem in this commit.  Let's address that as part of #2005.